pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -16,6 +16,9 @@
 from datetime import datetime, timedelta
 from utils import getSeconds
 
+from pprint import pprint
```

Comment:
Was used during development and does not hurt as it is very useful for future development.